frontend/src/services/http/http.service.ts

```diff
@@ -47,6 +47,7 @@ class Http {
     if (!response.ok) {
       throw new HttpError({
         status: response.status,
+        message: response.statusText,
```

Comment:
```suggestion
        message: response.statusText,
```
First you need to check `body.messages`, because it should be an array of error messages